hooks/useFeature.js

```diff
@@ -0,0 +1,197 @@
+        labelText
+        summary
+        coverImage {
+          sources {
+            uri
+          }
+        }
+        relatedNode {
+          id
+          ... on Url {
+            url
+          }
+          ... on ContentChannel {
+            name
+          }
+        }
+      }
+    }
+    ... on HeroListFeature {
+      id
+      title
+      subtitle
+      actions {
+        id
+        title
+        subtitle
+        action
+        image {
+          sources {
+            uri
+          }
+        }
+        relatedNode {
+          id
+          ... on Url {
+            url
+          }
+          ... on ContentChannel {
+            name
+          }
+        }
+      }
+      primaryAction {
+        title
+        action
+        relatedNode {
+          id
+          ... on Url {
+            url
+          }
+          ... on ContentChannel {
+            name
+          }
+        }
+      }
+      heroCard {
+        action
+        title
+        hasAction
+        actionIcon
+        labelText
+        summary
+        coverImage {
+          sources {
+            uri
+          }
+        }
+        relatedNode {
+          id
+          ... on Url {
+            url
+          }
+          ... on ContentChannel {
+            name
+          }
+        }
+      }
+    }
+    ... on HorizontalCardListFeature {
+      id
+      title
+      subtitle
+      cards {
+        action
+        title
+        hyphenatedTitle: title(hyphenated: true)
+        hasAction
+        actionIcon
+        labelText
+        summary
+        coverImage {
+          sources {
+            uri
+          }
+        }
+        relatedNode {
+          id
+          ... on Url {
+            url
+          }
+          ... on ContentChannel {
+            name
+          }
+        }
+      }
+      primaryAction {
+        title
+        action
+        relatedNode {
+          id
+          ... on Url {
+            url
+          }
+          ... on ContentChannel {
+            name
+          }
+        }
+      }
+    }
+  }
+}
+
+`;
+
+// export const GET_FEATURE = gql`
+//   query getFeature($featureId: ID!) {
+//     node(id: $featureId) {
+//       id
+//       __typename
+//       ... on VerticalCardListFeature {
+//         ...VerticalCardListFeatureFragment
+//       }
+//     }
+//   }
+
+//   fragment VerticalCardListFeatureFragment on VerticalCardListFeature {
+//     id
+//     isFeatured
+//     title
+//     subtitle
+//     cards {
+//       action
+//       title
+//       hasAction
+//       actionIcon
+//       labelText
+//       summary
+//       coverImage {
+//         sources {
+//           uri
+//         }
+//       }
+//       relatedNode {
+//         ...RelatedFeatureNodeFragment
+//       }
+//     }
+//   }
+
+//   fragment RelatedFeatureNodeFragment on Node {
+//     id
+//     ... on Url {
+//       url
+//     }
+//     ... on ContentChannel {
+//       name
+//   }
+
+// `;
```

Comment:
If we don't need this, let's just delete it in a single commit. Commented-out code can always be recovered if we have a single `Remove ...` commit message to find in the history.